modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledMenuItemTest.ts

```diff
@@ -0,0 +1,53 @@
+import { Keys, Mouse, RealKeys, UiFinder, Waiter } from '@ephox/agar';
+import { afterEach, beforeEach, describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import { Editor } from 'tinymce/core/api/PublicApi';
+import TablePlugin from 'tinymce/plugins/table/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledMenuItemTest', () => {
+
+  const verticalAlignMenuItemSelector = '[title="Vertical align"]';
+  const tableMenuItemSelector = '[role="menuitem"]:contains("Table")';
+
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    plugins: 'table',
```

Comment:
We generally try to avoid pulling in plugins in silver tests unless absolutely necessary. For this case it's not necessary, so can we please either move this test to the table plugin or just register our own nested menu and do assertions against that (preferably the latter option).

Reply:
To provide more context, it's not the table plugin logic we want to test from the theme perspective but instead the fact that a nested menu won't open when it's disabled. This can also help with maintainability, as if we refactor how the table plugin works then we have less places we have to make updates to (ie it's largely all isolated inside the table plugin).

Reply:
Yep, makes sense 👍